### DISCUSSION TITLE
Read function output issue

### DISCUSSION THREAD

**Discussion by agganitk** [simpleble/simpleble#376] [Q&A]
while True:
     print(peripheral.read(service_uuid, characteristic_uuid)) 

 keeps on printing the last received string again and again until new string is received.  Once new string is received, it keeps on printing the new one again and again.  I want to print received text only once.

**Comment by kdewald** [comment]
Hey @agganitk, this could be a bug. Could you create an issue and share more information about the platform you're using?

Thanks!
